Docs/pagedeployment.md

```diff
@@ -0,0 +1,32 @@
+# ELEMWAVE PAGE DEPLOYMENT
+In order to deploy a public web site using GitHub pages we have to follow this steps:
+
+    1. First of all we have to create a repository where we will keep our web site configuration and programmation (It has to be public if you want to use the free version of Git pages).
```

Comment:
Programmation sounds weird, not sure if it's even a word. Code is probably what you wanted to say